Formula/s/slurm.rb

```diff
@@ -20,15 +20,12 @@ class Slurm < Formula
 
   depends_on "meson" => :build
   depends_on "ninja" => :build
-
-  uses_from_macos "ncurses"
+  depends_on "ncurses"
```

Comment:
Why?

Reply:
The Sonoma builds failed without that change. Everything else was successful.

Reply:
Can we make it depend on Sonoma then?

Reply:
FWIW, I wasn't able to reproduce this locally on Sonoma. But scoping to just Sonoma makes sense if we are moving forward.

Reply:
yeah, we can just scope this change ot sonoma builds, but I think the `ncurses 5` has lots of security bugs.

Reply:
@p-linnane I think it might be because of you already had `ncurses 6` installed?

Reply:
actually uninstalled `ncurses` also worked for me with a sonoma-arm machine. weird.

Reply:
>  but I think the ncurses 5 has lots of security bugs.

macOS doesn't ship with version 5, at least not on Ventura.